.buildscript/deploy_snapshot.sh

```diff
@@ -6,7 +6,7 @@
 # http://benlimmer.com/2013/12/26/automatically-publish-javadoc-to-gh-pages-with-travis-ci/
 
 SLUG="square/javapoet"
-JDK="oraclejdk8"
+JDK="oraclejdk7"
```

Comment:
That occurring would be a bug in `javac` I think? For the last year every single release I've made of this library, all of Square's that I touch, and my personal libraries has been made with JDK 8. Yes I do it because I'm vain and want the pretty Javadoc, but I've always relied on the `-target 1.7` to save us.